sauce/features/accounts/customFlagNames/index.js

```diff
@@ -0,0 +1,148 @@
+      $('.ynab-flag-yellow .label, .ynab-flag-yellow .label-bg').html(yellowFlagLabel);
+      $('.ynab-flag-green .label, .ynab-flag-green .label-bg').html(greenFlagLabel);
+      $('.ynab-flag-purple .label, .ynab-flag-purple .label-bg').html(purpleFlagLabel);
+
+      $('.modal-account-flags .modal').css({ height: '22em' })
+        .append('<div id="account-flags-actions" style="padding: 0 .3em"><button id="flags-edit" class="button button-primary">Edit <i class="flaticon stroke compose-3"><!----></i></button></div>');
+
+      this.addEventListeners();
+    }
+  }
+
+  onRouteChanged() {
+    if (!this.shouldInvoke()) return;
+
+    this.invoke();
+  }
+
+  addEventListeners() {
+    let $this = this;
+    $('#flags-edit').click(function () {
+      $('.modal-account-flags .modal-list').empty();
+
+      for (let key in flags) {
+        let flag = flags[key];
+        $('.modal-account-flags .modal-list').append('<li><input type="text" id="' + key + '" class="flag-input" style="color: #fff; fill: ' + flag.color + '; background-color: ' + flag.color + '; height: 30px; padding:0 .7em; margin-bottom: .3em; border: none;" value="' + flag.label + '" placeholder="' + flag.label + '" /></li>');
+      }
+
+      $('#account-flags-actions').empty();
+
+      $('#account-flags-actions').append('<button id="flags-close" class="button button-primary">Close <i class="flaticon stroke checkmark-2"><!----></i></button>');
+
+      $('input.flag-input').focus(function () {
+        $(this).css({
+          color: '#000'
+        });
+      });
+
+      $('input.flag-input').blur(function () {
+        $(this).css({
+          color: '#fff'
+        });
+        $this.saveFlag($(this));
+      });
+
+      $('#flags-close').click(function () {
+        $('.modal-overlay').click();
+      });
+    });
+  }
+
+  saveFlag(flag) {
+    if (flag.attr('placeholder') !== flag.val()) {
+      let key = flag.attr('id');
+      flags[key].label = flag.val();
+      localStorage.setItem('flags', JSON.stringify(flags));
+      this.updateFlagLabels();
+      this.invoke();
+    }
+  }
+
+  updateFlagLabels() {
+    redFlagLabel = flags.red.label;
+    blueFlagLabel = flags.blue.label;
+    orangeFlagLabel = flags.orange.label;
+    yellowFlagLabel = flags.yellow.label;
+    greenFlagLabel = flags.green.label;
+    purpleFlagLabel = flags.purple.label;
+  }
+
+  storeDefaultFlags() {
+    const flagsJSON = {
+      red: {
+        label: 'Red',
+        color: '#d43d2e'
+      },
+      orange: {
+        label: 'Orange',
+        color: '#ff7b00'
+      },
+      yellow: {
+        label: 'Yellow',
+        color: '#f8e136'
+      },
+      green: {
+        label: 'Green',
+        color: '#9ac234'
+      },
+      blue: {
+        label: 'Blue',
+        color: '#0082cb'
+      },
+      purple: {
+        label: 'Purple',
+        color: '#9384b7'
+      }
+    };
+    localStorage.setItem('flags', JSON.stringify(flagsJSON));
```

Comment:
so this is a new framework and unfortunately i haven't gotten to adding the localstorage methods yet :(

you could add them or just prefix these settings with `ynab-toolkit-`

the old ones are here: https://github.com/toolkit-for-ynab/toolkit-for-ynab/blob/master/source/common/res/features/shared/main.js#L328-L340

if you wanted to add them to the new toolkit helper you can throw them in here: https://github.com/toolkit-for-ynab/toolkit-for-ynab/blob/master/sauce/helpers/toolkit.js